src/components/App/index.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div id="app" class="App">
+  <!-- FIXME: only do this on iOS -->
+  <div id="app" class="App" @click="iOSFix()">
```

Comment:
This needs to only be run for iOS. It's not necessary on desktop / android. I just put it in there as a quick fix and didn't have time to make it iOS-only